ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -106,26 +107,46 @@
                                             update-type)]))))
 
 (defn- get-provider-concept-ids
-  "Returns a list of collection concept ids for a given provider-id"
+  "Returns a list of collection concept ids for a given provider-id.
+   Throws exception if non are returned."
   [context provider-id]
   (let [concepts (mdb/find-concepts context 
```

Comment:
Since we can only update the latest revision of a collection, we should use `latest = true` to  only get the latest revision back. Also, I think this call will return tombstones (not certain) which we will want to skip.

Reply:
Done. I checked the metadata db call, it didn't exclude any deleted ones.